packages/tokamak/sdk/tasks/approve-deposit-ton.ts

```diff
@@ -94,17 +94,17 @@ const erc20ABI = [
 
 const zeroAddr = '0x'.padEnd(42, '0')
 
-let TON = process.env.TON || ''
+let l2NativeToken = process.env.NATIVE_TOKEN || ''
 let addressManager = process.env.ADDRESS_MANAGER || ''
 let l1CrossDomainMessenger = process.env.L1_CROSS_DOMAIN_MESSENGER || ''
 let l1StandardBridge = process.env.L1_STANDARD_BRIDGE || ''
 let optimismPortal = process.env.OPTIMISM_PORTAL || ''
 let l2OutputOracle = process.env.L2_OUTPUT_ORACLE || ''
 
 const updateAddresses = async (hre: HardhatRuntimeEnvironment) => {
-  if (TON === '') {
-    const Deployment__TON = await hre.deployments.get('L2NativeToken')
-    TON = Deployment__TON.address
+  if (l2NativeToken === '') {
```

Comment:
I think this task is for specific TON

Reply:
@nguyenzung I just renamed it for matching with other tasks, use the term "l2NativeToken" for "TON"

Reply:
ApproveAndCall is used for TON, it is not a standard function for other tokens. I think it is still OK of changing the name. Anw, I still found some TON in the code. I think we should update them.